libdnf/rpm/solv_sack.cpp

```diff
@@ -115,24 +116,18 @@ void libsolv_repo_free(LibsolvRepo * libsolv_repo) {
 // only works if there are no duplicates both in q1 and q2
 // the map parameter must point to an empty map that can hold all ids
 // (it is also returned empty)
-int is_superset(Queue * q1, Queue * q2, Map * m) {
+int is_superset(solv::IdQueue & q1, Queue * q2, solv::SolvMap & map) {
```

Comment:
Queues can be marked as `const`:
`int is_superset(const solv::IdQueue & q1, const Queue * q2, solv::SolvMap & map) {`